File: main/public/problems/python/python_files/num4.py
n = int(input("Enter a number: "))

sum = 0
for i in range(n + 1):
    if i % 3 == 0 or i % 5 == 0:
        print(i)
        sum += i
        
print(f"The sum of numbers divisible by 3 or 5 up to {n} is: {sum}")

File: main/public/problems/cpp/cpp_files/num4.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cout << "Enter a number: ";
    cin >> n;

    int sum = 0;
    for (int i = 1; i <= n; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            sum += i;
        }
    }

    cout << "The sum of numbers divisible by 3 or 5 up to " << n << " is: " << sum << endl;
    return 0;
}
